superset-frontend/src/explore/components/PropertiesModal.tsx

```diff
@@ -249,10 +250,17 @@ function PropertiesModal({ slice, onHide, onSave }: InternalProps) {
         </Row>
       </Modal.Body>
       <Modal.Footer>
-        <Button type="button" buttonSize="sm" onClick={onHide} cta>
+        <Button
+          data-test="cancel-button"
```

Comment:
With so many cancel buttons in Superset, do you think this may benefit from being more specific, e.g. `properties-modal-cancel-button`? Might not be a big deal with some of these, but I'm thinking it's better to err on the side of specificity, so we know it's the _right_ cancel button (or whatever) in longer, more complex tests.